src/opnsense/scripts/netflow/lib/parse.py

```diff
@@ -40,11 +41,11 @@ def __init__(self):
         """ construct local interface mapping
         """
         self._if_index = dict()
-        sp = subprocess.run(['/sbin/ifconfig', '-l'], capture_output=True, text=True)
-        if_index = 1
-        for line in sp.stdout.split():
-            self._if_index["%s" % if_index] = line
-            if_index += 1
+        sp = subprocess.run(['/usr/local/sbin/ifinfo'], capture_output=True, text=True)
```

Comment:
@AdSchellevis does this look good for you or have a simplification in mind?

Reply:
@fichtner looks good to me